src/__tests__/commands/utilities/ping.test.ts

```
@@ -0,0 +1,395 @@
+/**
+ * Test Suite: Ping Command
+ *
+ * Comprehensive tests for the ping command module.
+ * Tests cover command structure, API latency calculation, WebSocket heartbeat retrieval,
+ * embed formatting, and error handling scenarios.
+ */
+
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { SlashCommandBuilder, EmbedBuilder, ChatInputCommandInteraction } from 'discord.js';
```

ESLint

[error] 'SlashCommandBuilder' is defined but never used.

(@typescript-eslint/no-unused-vars)